patroni/postgresql.py

```diff
@@ -59,7 +60,8 @@ def __init__(self, config):
         self.callback = config.get('callbacks') or {}
         self.use_slots = config.get('use_slots', True)
         self._schedule_load_slots = self.use_slots
-        self._postgresql_conf = os.path.join(self._data_dir, 'postgresql.conf')
+        self._postgresql_conf = os.path.join(self._data_dir,
+                                             config.get('config', 'postgresql.conf'))
```

Comment:
Oh. May be you didn't noticed, but recently we had changed how Patroni works with config files.
After initializing of new database it renames ``postgresql.conf`` into ``postgresql.base.conf``.
Later it writes into ``postgresql.conf`` something like:
```
include 'postgresql.base.conf'
param1 = 'value1'
param2 = 'value2'
etc...
```

In your case it will rename ``pipelinedb.conf`` into ``postgresql.base.conf``. It will not hurt, but doesn't look nice.

I would propose rename ``config`` into ``config_base_name`` and at the end to have something like:
```diff
@@ -59,8 +59,9 @@ class Postgresql(object):
         self.callback = config.get('callbacks') or {}
         self.use_slots = config.get('use_slots', True)
         self._schedule_load_slots = self.use_slots
-        self._postgresql_conf = os.path.join(self._data_dir, 'postgresql.conf')
-        self._postgresql_base_conf_name = 'postgresql.base.conf'
+        config_base_name = config.get('config_base_name', 'postgresql')
+        self._postgresql_conf = os.path.join(self._data_dir, config_base_name + '.conf')
+        self._postgresql_base_conf_name = config_base_name + '.base.conf'
         self._postgresql_base_conf = os.path.join(self._data_dir, self._postgresql_base_conf_name)
         self._recovery_conf = os.path.join(self._data_dir, 'recovery.conf')
         self._configuration_to_save = (self._postgresql_conf, self._postgresql_base_conf,
```